### PR TITLE
Update readme with missing set up instructions

### PR DESCRIPTION
**Pull request by camccar**
Did not see anything about generating the rsa key so I am adding a step for that. The app will not start without it. ~~Also throwing in `npm install` for the ui portion.~~

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks so much for the contribution! We'll get this reviewed and merged soon. Can we show you some [appreciation](https://lbry.com/faq/appreciation)?

File: readme.md [hunk 1]
```diff
@@ -53,24 +53,32 @@ You still might want to use `docker` and `docker-compose` for running SDK and DB
...
 
 ```
 cd ..
 git clone git@github.com:lbryio/lbry-desktop.git
 ```
 
-**5. Launch UI in lbry-desktop repo folder**
+**5. Install npm dependencies**
+
+`npm install`
```
Comment (andybeletsky) [inline readme.md:73]: It's `yarn` as per [lbrytv-desktop readme suggestions](https://github.com/lbryio/lbry-desktop#prerequisites). That's why I think it's best to omit initialization steps for external repos from the readme.

Comment (camccar) [reply readme.md]: Good point. updated to remove that step. F

**Comment by andybeletsky** [comment]
Thank you @camccar!

**Pull request merged**